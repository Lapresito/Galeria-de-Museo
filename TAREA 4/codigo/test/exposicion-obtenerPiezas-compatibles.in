crearExposicion 10/05/2024 25/05/2024 50 1
obtenerPiezasExposicion
agregarAExposicion 8 0 1 2 3 4 5 6 7
obtenerPiezasExposicion
agregarAExposicion 7 5 6 7 8 9 10 11
obtenerPiezasExposicion
sonExposicionesCompatibles 08/05/2024 28/05/2024 50 2 1 1
sonExposicionesCompatibles 08/05/2024 28/05/2024 50 2 1 40
sonExposicionesCompatibles 15/05/2024 20/05/2024 50 2 1 1
sonExposicionesCompatibles 15/05/2024 20/05/2024 50 2 1 40
sonExposicionesCompatibles 05/05/2024 15/05/2024 50 2 1 1
sonExposicionesCompatibles 05/05/2024 15/05/2024 50 2 1 40
sonExposicionesCompatibles 15/05/2024 28/05/2024 50 2 1 1
sonExposicionesCompatibles 15/05/2024 28/05/2024 50 2 1 40
sonExposicionesCompatibles 02/05/2024 05/05/2024 50 2 1 1
sonExposicionesCompatibles 02/05/2024 05/05/2024 50 2 1 40
sonExposicionesCompatibles 27/05/2024 28/05/2024 50 2 1 1
sonExposicionesCompatibles 27/05/2024 28/05/2024 50 2 1 40
liberarExposicion
Fin