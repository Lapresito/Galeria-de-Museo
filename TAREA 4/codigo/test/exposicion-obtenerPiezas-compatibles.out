Bienvenido al programa principal de ArteProg. Por favor ingrese su comando:
1> Exposicion creada con exito.
2> Las piezas de la exposicion son las siguientes:

3> Piezas agregadas a la exposicion de forma exitosa.
4> Las piezas de la exposicion son las siguientes:
0 1 2 3 4 5 6 7 
5> Piezas agregadas a la exposicion de forma exitosa.
6> Las piezas de la exposicion son las siguientes:
0 1 2 3 4 5 6 7 8 9 10 11 
7> Las exposiciones NO son compatibles.
8> Las exposiciones son compatibles.
9> Las exposiciones NO son compatibles.
10> Las exposiciones son compatibles.
11> Las exposiciones NO son compatibles.
12> Las exposiciones son compatibles.
13> Las exposiciones NO son compatibles.
14> Las exposiciones son compatibles.
15> Las exposiciones son compatibles.
16> Las exposiciones son compatibles.
17> Las exposiciones son compatibles.
18> Las exposiciones son compatibles.
19> Exposicion liberada con exito.
20> Fin.
